src/components/form/form.scss

```diff
@@ -136,3 +136,76 @@
     padding-left: 1rem;
     padding-top: 0.25rem;
 }
+
+.mdc-typography--headline1 {
+    position: relative;
+}
+
+.form-group {
+    .mdc-typography--headline1 {
+        --mdc-typography-headline1-font-size: 2rem;
+        --mdc-typography-headline1-line-height: 2.25rem;
+        --mdc-typography-headline1-letter-spacing: -0.0625rem;
+        --mdc-typography-headline1-font-weight: 400;
+        color: rgb(var(--contrast-1000));
```

Comment:
If you want to make it great for dark-mode while you're at it, I think they need a stronger contrast in dark-moder…

Reply:
I think the color here is good. We probably have to reduce the grayness of the admin-background. e.g. use `--contrast-200` there.

Reply:
I only tested it in the docs, so I don't know if it's different in Lime Admin. But I don't think we have dark-mode in Lime Admin anyway, do we?

Light-mode:

![image](https://user-images.githubusercontent.com/154725/142418654-cb9480ba-4b1e-425f-af64-3b39fade6489.png)

Dark-mode:

![image](https://user-images.githubusercontent.com/154725/142418702-9c29cb0b-9485-4746-b7e5-8122170a2b93.png)

Now, obviously there are lots of things not supporting dark-mode in that screenshot, but I'm just looking at the contrast of the text vs the background. But I'm really not saying we need to fix that now. I just wanted to give you a chance to do it before I merged, if that's what you wanted to do. I'll rebase this branch. If you haven't said anything by the time it's built again, I'll merge it 😊